### PR TITLE
Initial tweaks

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

File: src/github.com/kegsay/gitpher/github_api.go [hunk 1]
```diff
@@ -5,21 +5,20 @@ import (
...
+func GetGithubUserContent(contentPath string) ([]byte, error) {
+	githubURL := githubUserContentURL + contentPath
 	response, err := http.Get(githubURL)
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 
 	defer response.Body.Close()
 	body, err := ioutil.ReadAll(response.Body)
```
Comment (kegsay) [inline src/github.com/kegsay/gitpher/github_api.go:19]: Probably worth at this point just returning this function?

Comment (illicitonion) [inline src/github.com/kegsay/gitpher/github_api.go:19]: It's a trade-off, right? I don't know the implementation details of HttpResponse.Body, but my guess is that either you leave the TCP socket open and unread (by returning response.Body), or you keep the whole response in memory (by reading to a []byte).

Comment (kegsay) [inline src/github.com/kegsay/gitpher/github_api.go:19]: I mean `return ioutil.ReadAll(response.body)` rather than `return response.Body`.

**Comment by kegsay** [comment]
Thanks for the tweaks :)

File: src/github.com/kegsay/gitpher/github_api.go (hunk 1 above)
Comment (illicitonion) [inline src/github.com/kegsay/gitpher/github_api.go:19]: Oh, yeah, sure :)

**Pull request merged**